atd-vzd/migrations/migration_crashes_2021_07_15--1355.sql

```diff
@@ -0,0 +1,2 @@
+-- Add a new column to the atd_txdot_crashes table to hold crash basaed comprehensive cost
+ALTER TABLE atd_txdot_crashes ADD COLUMN est_comp_cost_crash_based numeric(10,2) default 0;
```

Comment:
Defaults to zero because the amount is not known. The trigger will update this column.